trento/xml/article_sap_trento.xml

```diff
@@ -780,12 +780,15 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
   -->
   <section xml:id="sec-trento-checks">
     <title>Performing configuration checks</title>
+    <remark>toms 2024-04-18 (change#8): First saptune related configuration checks (host and cluster specific) -> update</remark>
     <para>
-      One of Trento's main features is to provide checks for the HANA scale-up performance optimized two-node clusters in your SAP Landscape.
-      To run these checks, proceed as follows:
+     One of Trento's main features is to provide configuration checks that ensure your infrastructure setup adheres to our Best Practices, 
+     or other vendor's Best Practices, and does not drift away in time. Configuration checks ara available for
+     HANA clusters, ASCS/ERS clusters and hosts. The following procedure is specific to a HANA cluster. The procedure for an ASCS/ERS cluster
+     or a host would be exactly the same, only starting from the corresponding Details view:
```

Comment:
Some typos and split some sentence. Check if it's okay:

```suggestion
     One of Trento's main features is to provide configuration checks that ensure your infrastructure setup adheres to best practices from you or other vendors.
     This ensure that your infrastructure setup does not drift away in time. Configuration checks are available for
     HANA clusters, ASCS/ERS clusters and hosts. The following procedure is specific to a HANA cluster. The procedure for an ASCS/ERS cluster
     or a host would be exactly the same, only starting from the corresponding Details view:
```

Reply:
A couple of change requests here @tomschr:

- "our Best Practices" is intentional. It refers to SUSE's best practices. So I suggest the following: "One of Trento's main features is to provide configuration checks that ensure your infrastructure setup adheres to best practices from SUSE or other vendors."
- "This ensures" instead of "This ensure".

Reply:
Ahh, I thought best practices refers to general methodology, not specifically for SUSE. Now it makes sense. Thank you!

In that case, you should use the name "SUSE Best Practices" (that's the official name). I would also slightly rephase it like this:

```
One of Trento's main features is to provide configuration checks that ensure
your infrastructure setup follows <link xlink:href="https://documentation.suse.com/sbp-supported.html">SUSE Best Practices</link>
or recommendations from other vendors.
```